chapters/simple_exec.md

```diff
@@ -17,44 +17,36 @@ metas:
   name: clear
 requests:
   on:
-    [SearchRequest, TrainRequest, IndexRequest]:
-      - !ReqPruneDriver {}
+    SearchRequest:
+      - !ExcludeReqQL
```

Comment:
are u sure this is the name of a Driver?

Reply:
https://github.com/jina-ai/jina/blob/master/jina/resources/executors._clear.yml

Reply:
I think it's this one?

Reply:
This YAML file is also updating frequently, so we'd better guide the location of these execs